substrate/frame/revive/Cargo.toml

```diff
@@ -69,6 +69,7 @@ subxt-signer = { workspace = true, optional = true, features = ["unstable-eth"]
 alloy-consensus = { workspace = true, default-features = true }
 array-bytes = { workspace = true, default-features = true }
 assert_matches = { workspace = true }
+ethereum = { version = "0.18.2", default-features = false, features = ["with-scale"] }
```

Comment:
Do we need this?

Reply:
Well, I wanted to use another implementation to cross-check rlp encoding/decoding.

It was used in below test:
https://github.com/paritytech/polkadot-sdk/blob/abdcfb43b93b6f482b97bef61716d4b3b5061c56/substrate/frame/revive/src/evm/api/rlp_codec.rs#L717-L731

Actually, i think there's no need for such additional cross-check. I will remove it.